### DISCUSSION TITLE
System-wide shortcut keys no longer works after updating to 4.2.0-RC

### DISCUSSION THREAD

**Discussion by superluminal300** [OnionUI/Onion#1018] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

Onion V4.2.0-RC

### Miyoo Firmware Version

20230609

### Provide a clear and concise description of the issue

Upon update of Miyoo Mini Plus from Onion V4.2.0-beta-dev to Onion V4.2.0-RC, the Global Shortcuts stopped working per listed here: https://github.com/OnionUI/Onion/wiki/Features

RetroArch main menu can no longer be called up with Menu+Select, and none of the other shortcuts functions as well.

### Steps to Reproduce

Update from Onion V4.2.0-beta-dev to Onion V4.2.0-RC

### Provide any additional context or information that may help us investigate your issue

Upon update, there was a message that retroarch config was reset. Perhaps this has something to do with the problem?

**Comment by Aemiii91** [comment]
Try resetting everything in `Tweaks -> Advanced -> Reset settings`
